src/orca/services/synapse/ops.py

```diff
@@ -57,3 +58,73 @@ def monitor_evaluation_queue(self, evaluation_id: str) -> bool:
         )
         submissions_num = sum(1 for submission in received_submissions)
         return submissions_num > 0
+
+    def trigger_indexing(self, syn, submission_view) -> None:
```

Comment:
Nit: this is a general function for views (submission, file, entity, etc).  Type hinting.